frontend/src/pages/ProjectOverviewPage.tsx

```diff
@@ -22,11 +22,21 @@ export const ProjectOverviewComponent = () => {
     roadmapsSelector,
     shallowEqual,
   );
+  const [sortedRoadmaps, setSortedRoadmaps] = useState<Roadmap[] | undefined>(
+    [],
+  );
 
   useEffect(() => {
     if (!roadmaps) dispatch(roadmapsActions.getRoadmaps());
   }, [dispatch, roadmaps]);
 
+  useEffect(() => {
+    if (roadmaps)
+      setSortedRoadmaps(
+        [...roadmaps].sort((a, b) => a.name.localeCompare(b.name)),
+      );
+  }, [roadmaps]);
+
```

Comment:
Im not sure if useState and useEffect are even needed here, probably the only thing re-rendering the component would be the roadmaps changing and the selector triggering. But either way is fine with me
```
const roadmaps = useSelector<RootState, Roadmap[] | undefined>(
    roadmapsSelector,
    shallowEqual,
  );
const sortedRoadmaps = [...roadmaps].sort((a, b) => a.name.localeCompare(b.name)),
```